### ISSUE TITLE
Help correct the MARTINGALE of this bot

### ISSUE THREAD

**Issue by nobleb2007** [Fruitfella/ROBOT-binary#406]
Please, help me correct the MARTINGALE of this bot, is not martingale and showing slice e error and stop when supposed to MARTINGALE the loss and return to original amount for trading. Please help my fix the bot
@Fruitfella 
@delano24
@zdeglet
@sinung21
[HLBN001_V100_DT10_Higher_Lower_Bot.txt](https://github.com/Fruitfella/ROBOT-binary/files/3123651/HLBN001_V100_DT10_Higher_Lower_Bot.txt)
Please help out.
God bless you all.
@AlienFly
@ valentino242
@ubanemas
@okovalov
@ gokula-krishna-dev

**Comment by ZDEGLET** [comment]
пришли исходный робот

**Comment by nobleb2007** [comment]
[paid bot (1).txt](https://github.com/Fruitfella/ROBOT-binary/files/3124117/paid.bot.1.txt)

This is the bot @ZDEGLET

**Comment by ZDEGLET** [comment]
Я понимаю что мартингейл не работает. Нужно чтоб заработал?

**Comment by nobleb2007** [comment]
Yes, martingale does not work, I want the bot to work correctly with MARTINGALE.

**Comment by nobleb2007** [comment]
Please, help me correct the MARTINGALE error from this bot.

Please

**Comment by ZDEGLET** [comment]
У МЕНЯ НИЧЕГО НЕ ВЫХОДИТ СЛИШКОМ ЕГО ЗАМУДРИЛИ. Я ВИЖУ ЧТО НА @Fruitfella ЗА НЕГО НИКТО НЕ ВЗЯЛСЯ. Я ЕЩЕ ЕГО СМОТРЮ. ДАЖЕ НЕ ЗНАЮ КАК ЕГО ЕЩЕ ПЕРЕВЕРНУТЬ. РАБОТАЕМ. МНЕ САМОМУ ИНТЕРЕСНО ЕГО ВЗОРВАТЬ. ДО ПЕРВОГО ПРОИГРЫША ОН РАБОТАЕТ А ПОТОМ ПЕРЕСТАЕТ. НУЖНО В 4 БЛОКЕ ВЫСТАВИТЬ ФУНКЦИЮ ЧТОБ НАЧАЛ РАБОТАТЬ КАК ВНАЧАЛЕ. ПОКА НЕ ВЫХОДИТ.
ТЫ ГДЕ ЕГО ОТКОПАЛ ЭТО ЧУДО?

**Comment by nobleb2007** [comment]
I don't understand what you meant even after translate in Google. Am sure you meant well. 

Please anybody in the house can help me to correct the MARTINGALE of this bot.

I will be very grateful.
Thanks

**Comment by nobleb2007** [comment]
Can somebody help please with the correction of my bot on MARTINGALE.

Thanks

**Comment by ZDEGLET** [comment]
[paid.bot.1 (1).txt](https://github.com/Fruitfella/ROBOT-binary/files/3153679/paid.bot.1.1.txt)
кто то его сделал только немного другой
